samNode/layers/reservationLayer/reservationLayer.js

```diff
@@ -1,7 +1,7 @@
 const { TABLE_NAME, TIMEZONE, runQuery, getOne, logger, marshall, dynamoClient, PutItemCommand, unmarshall, UpdateItemCommand } = require('/opt/baseLayer'); 
 const { DateTime } = require('luxon');
 // TODO: provide these as vars in Parameter Store
-const LOW_CAPACITY_THRESHOLD = process.env.LOW_CAPACITY_THRESHOLD || 0.25;
+const LOW_CAPACITY_THRESHOLD = process.env.LOW_CAPACITY_THRESHOLD || 0.25; 
```

Comment:
Remove space inserted here.

Reply:
fixed in [commit](https://github.com/bcgov/parks-reso-api/pull/446/commits/44ca768f59f5208cf87b0b6280be06a368b0075e)